lang/python/python3-sqlparse/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sqlparse
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+PKG_SOURCE:=sqlparse-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/sqlparse
+PKG_HASH:=7c3dca29c022744e95b547e867cee89f4fce4373f3549ccd8797d8eb52cdb873
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/sqlparse-$(PKG_VERSION)
```

Comment:
It shouldn't be necessary to override default settings.

Reply:
Thought it would be good to have it in sqlparse-x.y.z instead of sqlparse-python3/...